### PR TITLE
Added Docker image build to CI

### PR DESCRIPTION
**Pull request by tk-nguyen**
We should add our Docker Hub credentials to secrets if we are using this. The secret keys should be `DOCKERHUB_USERNAME` and `DOCKERHUB_TOKEN`.

### CODE REVIEW

**Review by hoangdat — APPROVED** [review]

File: .github/workflows/image.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
+on:
+  push:
+    branches:
+      - "master"
```
Comment (hoangdat) [inline .github/workflows/image.yaml:4]: how about `release` branch?

Comment (tk-nguyen) [reply .github/workflows/image.yaml:4]: That's handled by the tags trigger below.

**Comment by hoangdat** [comment]
please make a move and remove WIP, Can we merge?

**Comment by tk-nguyen** [comment]
> please make a move and remove WIP, Can we merge?

Not yet, until the Docker Hub situation is resolved, we can move forward with a dedicated account for CI.

**Pull request merged**